contracts/token/ERC721/ERC721BasicToken.sol

```diff
@@ -38,7 +38,7 @@ contract ERC721BasicToken is SupportsInterfaceWithLookup, ERC721Basic {
 
   // Equals to `bytes4(keccak256("onERC721Received(address,address,uint256,bytes)"))`
   // which can be also obtained as `ERC721Receiver(0).onERC721Received.selector`
-  bytes4 private constant ERC721_RECEIVED = 0x150b7a02;
+  bytes4 internal constant ERC721_RECEIVED = 0x150b7a02;
```

Comment:
Can you clarify why you're making this variable `internal`? It's non-trivial because if I understand correctly it will now occupy a slot in storage.

Reply:
It's also the kind of change that shouldn't be done ad-hoc in openzeppelin-zos.

Reply:
Hmm I wasn' t aware of the fact that internal constants took a slot in storage. This was more a convenience for contracts that extended ERC721, but I agree that if it implies a change in the storage layout, we should not do the change this lightly.

Reply:
I had to check and I was mistaken. It does not occupy a slot. I still believe it's better not to make it internal here, to avoid diverging from openzeppelin-solidity too much.

Reply:
Ok, got it. I'll make a note to make it internal in oz-sol.